### DISCUSSION TITLE
High Level Control of Robots

### DISCUSSION THREAD

**Discussion by Jouiii** [isaac-sim/IsaacLab#299] [Q&A] [answered]
Hello all
I would like to control the robots directly at a higher level and not just at the joint level. My goal is to control a quadruped or a drone with twist commands. I've already tried to use the [Anymal Controller](https://docs.omniverse.nvidia.com/py/isaacsim/source/extensions/omni.isaac.quadruped/docs/index.html#omni.isaac.quadruped.robots.Anymal) to derive the joint angles using the advance() method and then pass them to the ActionManager, but then I ran into problems with numpy arrays (switching to the CPU is not my preferred solution in the long run). 
I saw that #260 goes in a similar direction with the action graph, but I'm more interested in a coding solution. So, to summarize: how do you use complex mobile robots at a higher level of abstraction in Orbit?

Thanks in advance for any help.

**Comment by Mayankm96** [comment, marked as answer]
We have this feature under review. Should be merged next week :)

**Reply by Jouiii** [reply]
Great news and thanks for the reply.

**Reply by shiven2001** [reply]
Did this feature end up getting added? If yes, where can I find it?
